books/RayTracingTheRestOfYourLife.html

```diff
@@ -2531,7 +2531,7 @@
 This yields an exactly matching result so all we’ve done so far is move some computation up into the
 `cosine_pdf` class:
 
-  ![Image 9: Cornell box with a cosine density _pdf_
+  ![Image 9: Cornell box with a cosine density PDF
   ](../images/img-3.06-cornell-cos-pdf.jpg class=pixel)
```

Comment:
Can you rename the images in book 3 to be sequential (i.e. match the `Image #` with the `img-3.##`) ?

Reply:
Hmmm, image 4 and image 5 both use `img-3.02-cornell-refactor2.jpg`. Note sure how to untangle.